tools/__tasks__/compile/javascript/babel.js

```diff
@@ -0,0 +1,12 @@
+const execa = require('execa');
+
+const { src, transpiled } = require('../../config').paths;
+
+module.exports = {
+    description: 'Transpile',
+    task: () => execa('babel', [`${src}/javascripts`, '--out-dir', `${transpiled}/javascripts`, '--ignore', 'eslintrc.js'], {
```

Comment:
You ignore `eslintrc.js` in both babel commands, can you not do it in the `babelrc`?